includes/wikia/rabbitmq/ConnectionBase.class.php

```diff
@@ -43,19 +45,34 @@ public function publish( $routingKey, $body ) {
 			);
 
 			$channel->wait_for_pending_acks( self::ACK_WAIT_TIMEOUT_SECONDS );
+			ServiceFactory::instance()->rabbitFactory()->getCircuitBreaker()->SetOperationStatus( true );
```

Comment:
`SetOperationStatus` -> `setOperationStatus` throughout PR.

Reply:
Fix in 0e58d035787